gci/management/commands/gci_no_issue_tasks.py

```diff
@@ -0,0 +1,31 @@
+import os.path
+
+from ruamel.yaml import YAML
+from gci.gitorg import get_issue
+
+from django.core.management.base import BaseCommand
```

Comment:
Can you rearrange the import a bit? Put `django` import into the second group and `gci` import to the third group.
```
Standard library imports

Third-party imports

Application-specific imports
```